test/__snapshots__/StatsTestCases.basictest.js.snap

```diff
@@ -1099,7 +1099,7 @@ chunk (runtime: main) trees.js (trees) 215 bytes [rendered]
 `;
 
 exports[`StatsTestCases should print correct stats for ignore-warnings 1`] = `
-"asset main.js 1.37 KiB [emitted] (name: main)
+"asset main.js 989 bytes [emitted] (name: main)
```

Comment:
```bash
$ ls -la main.js                                                                                                                                                  
-rw-r--r--  1 ivankopeykin  staff  989 Feb  4 16:02 main.js
```
no `__webpack_modules__` now (all concatenated in one iife)

Reply:
That's not correct. This PR should only result in less inlining.

Reply:
with JavascriptModulesPlugin.js update this is correct one..